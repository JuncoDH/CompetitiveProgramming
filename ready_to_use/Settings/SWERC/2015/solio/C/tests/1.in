100
5
5 25 6 29 8
9
8 13 17 6 13 19 4 15 22
2
26 21
17
9 17 1 12 2 2 16 10 17 28 13 2 17 28 3 25 5
16
26 15 28 20 4 20 21 13 15 3 6 25 7 23 9 20
19
15 30 19 24 13 5 14 29 23 4 21 17 12 16 13 11 3 15 24
4
5 22 5 25
6
20 13 10 14 13 29
6
7 12 5 17 3 22
4
15 9 18 16
6
8 8 27 8 11 12
7
6 1 23 13 24 18 30
6
1 24 30 5 30 20
12
6 15 18 2 16 17 25 9 7 24 11 3
5
4 17 25 12 11
18
27 6 7 30 24 16 13 2 2 2 8 18 15 24 25 15 25 9
16
10 28 16 11 18 25 16 12 20 5 17 8 15 28 3 25
5
18 16 28 15 11
10
22 1 1 25 29 2 19 15 13 9
10
4 16 3 18 19 24 26 27 17 11
16
3 13 16 23 5 16 2 8 14 25 10 13 26 1 23 26
7
7 2 2 11 19 12 27
3
24 13 26
1
4
8
21 14 24 15 2 18 20 24
11
24 2 18 28 18 11 2 11 18 20 23
9
3 26 26 14 24 28 9 13 6
16
21 16 11 2 21 19 18 3 12 27 5 16 1 18 11 7
3
16 2 6
13
9 11 7 26 17 14 12 24 1 27 20 4 21
10
20 4 26 20 19 11 24 8 21 8
12
26 7 28 29 19 17 8 6 16 29 28 26
12
15 2 8 9 28 10 8 29 29 11 22 11
20
8 21 18 29 11 24 29 16 27 22 5 14 6 20 19 8 7 8 3 17
9
13 1 21 5 1 10 20 15 5
13
14 24 10 2 10 7 3 27 18 14 12 18 17
13
3 30 29 25 10 7 15 1 10 6 18 2 11
12
23 24 27 11 8 7 15 16 15 11 11 24
1
29
14
8 5 14 4 11 3 26 21 29 17 18 19 24 15
11
7 12 10 15 23 15 16 27 9 22 12
17
26 2 12 28 26 27 14 20 5 10 23 12 30 29 11 29 14
19
23 21 22 6 7 15 4 26 26 4 20 28 1 7 4 12 24 12 2
4
17 17 18 2
15
22 30 25 16 8 23 9 23 27 18 27 20 8 18 19
3
15 25 20
3
16 1 30
5
4 9 21 29 22
5
9 17 30 4 7
4
17 29 6 16
3
26 20 17
5
29 8 24 13 15
16
8 11 18 21 24 9 8 8 16 10 9 11 23 29 8 10
1
9
3
28 5 8
19
12 8 12 24 28 11 15 18 20 23 28 16 12 27 17 21 22 1 10
1
3
20
29 22 8 13 28 3 2 5 12 26 2 6 16 12 16 6 24 26 22 15
8
14 4 13 24 10 24 29 20
20
21 29 1 9 10 20 15 24 21 6 7 7 29 15 7 19 30 8 26 22
13
12 2 2 25 29 3 17 28 16 17 8 17 19
16
24 26 29 6 19 13 12 29 12 21 7 11 5 29 13 1
13
21 15 5 29 17 11 4 8 9 7 24 5 20
8
27 26 5 2 3 21 5 28
5
10 7 30 26 21
2
11 15
13
12 9 12 27 18 5 8 27 21 6 1 19 6
2
7 8
5
13 2 19 15 11
10
28 29 18 15 4 22 13 2 7 17
18
24 12 21 13 26 10 24 3 14 7 19 19 3 11 8 8 17 2
4
23 17 11 3
9
30 7 24 19 24 17 7 30 5
5
14 9 29 2 17
6
2 14 3 17 21 29
19
1 6 30 28 27 26 5 21 14 11 16 29 16 30 26 5 10 16 23
7
16 5 22 22 2 26 26
3
25 6 1
18
28 29 22 12 30 12 21 21 16 4 29 23 30 28 20 4 1 17
9
3 7 28 26 15 23 13 16 13
3
3 4 20
15
8 20 6 15 5 24 18 6 13 23 29 9 13 24 14
14
9 10 6 20 17 28 12 28 7 23 27 29 13 28
4
2 16 8 26
6
28 8 27 15 14 17
12
16 10 28 23 9 18 28 21 24 6 25 26
10
2 4 23 6 14 30 26 27 25 22
4
7 3 7 22
3
27 23 13
20
17 4 24 27 13 5 1 27 3 17 24 22 29 3 28 11 17 16 28 23
8
11 18 23 4 5 17 24 25
18
14 27 23 9 7 15 29 2 14 9 6 5 22 7 16 10 8 26
8
23 10 2 17 29 10 11 10
5
20 25 25 28 16
8
12 4 12 22 27 15 25 12
18
7 26 20 13 11 6 9 15 30 3 25 20 2 10 30 10 17 24
11
19 12 27 30 8 16 11 7 18 25 12
4
2 30 28 7
12
6 7 11 5 19 15 29 13 17 19 23 2
6
13 11 9 11 20 10
