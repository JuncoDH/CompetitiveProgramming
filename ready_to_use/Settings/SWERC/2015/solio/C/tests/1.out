147
357
47
767
977
1267
99
248
155
116
183
295
252
513
153
1048
992
202
398
520
834
197
100
0
406
570
446
782
32
671
514
824
615
1296
253
617
557
669
0
775
569
1257
1041
108
1129
95
64
183
135
129
100
199
865
0
54
1347
0
1193
399
1350
671
975
610
300
205
26
590
15
133
481
955
99
490
149
196
1385
312
39
1391
437
34
853
945
88
272
820
551
66
99
1387
361
984
316
264
375
1100
620
113
562
189
